R/wrappers.R

```diff
@@ -12,3 +12,5 @@ colnamesInt = function(x, cols, check_dups=FALSE) .Call(CcolnamesInt, x, cols, c
 coerceFill = function(x) .Call(CcoerceFillR, x)
 
 testMsg = function(status=0L, nx=2L, nk=2L) .Call(CtestMsgR, as.integer(status)[1L], as.integer(nx)[1L], as.integer(nk)[1L])
+
+setlevels = function(x, value) invisible( if (is.factor(x)) .Call(Csetlevels, x, as.character(value), unique(value)) )
```

Comment:
Does `.Call` return `x` itself? If not, it should (for consistency to other `set` functions)

Related, `!is.factor` case, if it doesn't error, should still return `x`.

Reply:
When `x` is not a factor, then it won't not set any levels, then I don't think it should return `x` like that. User will assume operation went successfully, why don't we raise an error?

Reply:
This is already the case. Please see test 12.